client/src/features/Event/New/components/QuestionItem.tsx

```diff
@@ -8,24 +6,37 @@ import { Switch } from '@/shared/components/Switch';
 import { Text } from '@/shared/components/Text';
 
 type Props = {
-  questionNumber: number;
+  orderIndex: number;
+  questionText: string;
+  isRequired: boolean;
   onDelete: () => void;
+  onChange: (updated: Partial<{ questionText: string; isRequired: boolean }>) => void;
```

Comment:
p2.
요렇게 사용도 가능할 것 같네용
```suggestion
  onChange: (updated: Partial<Pick<Question, 'questionText' | 'isRequired'>>) => void;
```

Reply:
Pick 사용하도록 수정했습니다
https://github.com/woowacourse-teams/2025-ah-madda/pull/163/commits/ed1ceb09575aea8d2dc09e0b0622b59d5d146651